tangelo/linq/tests/test_translator_circuit.py

```diff
@@ -180,6 +180,26 @@ def test_qiskit(self):
         sim_results = qiskit_simulator.run(translated_circuit).result()
         np.testing.assert_array_almost_equal(sim_results.get_statevector(translated_circuit), reference_big_msq, decimal=6)
 
+    @unittest.skipIf("qiskit" not in installed_backends, "Test Skipped: Backend not available \n")
+    def test_qiskit_multi_control(self):
+        from tangelo.linq import Gate, Circuit, translate_circuit
```

Comment:
All of these have been imported already at the beginning of the module, I think they can be removed (L25)